packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/TimelineRow.tsx

```diff
@@ -16,19 +16,19 @@ import * as React from 'react';
 
 import './TimelineRow.css';
 
-type TimelineRowProps = {
+type TTimelineRowProps = {
```

Comment:
Are leading `T`s for types being enforced along with `I`s for interfaces or was this just because you caught it?
If not, we should find a rule for that as well.